examples/cis2-wccd/src/lib.rs

```diff
@@ -1066,4 +1128,35 @@ mod tests {
         claim_eq!(host.state().balance(&TOKEN_ID_WCCD, &ADDRESS_0), Ok(400u64.into()));
         claim!(host.get_transfers().is_empty());
     }
+
+    #[concordium_test]
```

Comment:
Could we add a test that calls the migration function? It might not make sense in this contract, but perhaps one of the other examples.
Just something trivial to ensure that it works as expected 😊

Reply:
I have added a unit test for a successful upgrade